node/resolvers/Queries/Settings.ts

```diff
@@ -16,9 +17,14 @@ export const getAppSettings = async (_: any, __: any, ctx: Context) => {
 
   const app: string = getAppId()
 
-  const settings: any = await vbase.getJSON('b2b_settings', app).catch(() => {
+  const settings = (await vbase.getJSON('b2b_settings', app).catch(() => {
```

Comment:
we can consider removing this catch block since vbase.getJSON not usually goes wrong.